tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -55,4 +115,50 @@ public void chownRecursive() throws IOException, AlluxioException {
     owner = mFileSystem.getStatus(new AlluxioURI("/testDir/testFile")).getOwner();
     Assert.assertEquals("user2", owner);
   }
+
+  /**
+   * Run chown command to change owner and group of a path.
+   *
+   * @param command the command to run
+   * @param expectedReturnValue return value expected from running the command
+   * @param expectedCommandOutput command output expected from running the command
+   * @param expectedOwner expected owner of the path
+   * @param expectedGroup expected group of the path
+   * @throws Exception
+   */
+  private void runChownOwnerAndGroup(String[] command, int expectedReturnValue,
+      String expectedCommandOutput, String expectedOwner, String expectedGroup) throws Exception {
+    int ret = mFsShell.run(command[0], command[1], command[2]);
+    Assert.assertEquals(expectedReturnValue, ret);
+    Assert.assertTrue(mOutput.toString().contains(expectedCommandOutput));
+    checkPathOwnerAndGroup(command[2], expectedOwner, expectedGroup);
+  }
+
+  /**
+   * Check whether the owner and group of the path are expectedOwner and expectedGroup.
+   *
+   * @param path the path to check
+   * @param expectedOwner the owner that we expect to own the path
+   * @param expectedGroup the expected group of the path
+   * @throws Exception
+   */
+  private void checkPathOwnerAndGroup(String path, String expectedOwner, String expectedGroup)
+      throws Exception {
+    String currentOwner = mFileSystem.getStatus(new AlluxioURI(path)).getOwner();
+    String currentGroup = mFileSystem.getStatus(new AlluxioURI(path)).getGroup();
+    Assert.assertEquals(expectedOwner, currentOwner);
+    Assert.assertEquals(expectedGroup, currentGroup);
+  }
+
+  /**
+   * Set up a group mapping service for test with {@link IdentityUserGroupsMapping}.
+   */
+  private void setupTestGroupMappingService() {
+    clearLoginUser();
+    GroupMappingServiceTestUtils.resetCache();
+    Configuration.set(PropertyKey.SECURITY_GROUP_MAPPING_CLASS,
```

Comment:
Done. Actually I find that using Configuration.set even in a static method annotated with BeforeClass can cause problems. (Any explanation will help me understand why.) Therefore I think I'll stick to using ConfigurationRule.